kola/tests/kubernetes/basic.go

```diff
@@ -54,6 +54,8 @@ func init() {
 				ClusterSize: 0,
 				Platforms:   []string{"gce", "do", "aws", "qemu", "azure"}, // TODO: fix packet, esx
 				Distros:     []string{"cl"},
+				// incompatible with docker >=20.10
+				ExcludeChannels: []string{"alpha"},
```

Comment:
With the recent changes we now can do
```suggestion
				MinVersion:  semver.Version{Major: 2956},
```
where 2956 is larger than the currently released Alpha 2955.

(and same for the other test)

Reply:
Yes, EndVersion ;)

Reply:
Yes, that worked better than MinVersion 😄